rules/build-cross-platform-apps/rule.md

```diff
@@ -6,7 +6,6 @@ authors:
   - title: Matt Goldman
     url: https://www.ssw.com.au/people/matt-goldman
 related:
-  - use-net-maui
   - why-react-is-great
```

Comment:
Why is this rule related? I don't think we should refer to React in this rule.

Reply:
@matt-goldman It's mentioned on line 41... but please remove the related if you think it's not helpful